### PR TITLE
[Tests-Only] run API tests from oC 10 in CI

### PR DESCRIPTION
**Pull request by kiranparajuli589**
running oC 10 API tests in CI using ocis-server

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```
Comment (dpakach) [inline .idea/.gitignore]: do not commit this file

**Comment by kiranparajuli589** [comment]
I'm surprised, why http status changes from 204 to 201
Screenshot from 2020-03-25 17-12-50

**Comment by individual-it** [comment]
@kiranparajuli589 check the pulled in dependencies, maybe they are different versions in reva and here

File: .drone.star [hunk 2]
```diff
@@ -60,70 +102,95 @@ def testing(ctx):
...
+          'REVA_LDAP_HOSTNAME':'ldap',
+          'TEST_EXTERNAL_USER_BACKENDS':'true',
+          'TEST_OCIS':'true',
+          'OCIS_REVA_DATA_ROOT': '/srv/app/tmp/reva/'
+        },
         'commands': [
-          'make build',
+          'git clone -b master --depth=1 https://github.com/owncloud/core.git /srv/app/testrunner',
+          'cd /srv/app/testrunner',
+          'make test-acceptance-api'
```
Comment (kiranparajuli589) [inline .drone.star]:
```suggestion
          'make test-acceptance-api'
          'find /srv/app/tmp/reva/'
```

**Comment by phil-davis** [comment]
I wonder what can be done about the many messages in https://cloud.drone.io/owncloud/ocis/271/1/11 ocis-server log:
```
6:33AM WRN srv/app/pkg/mod/github.com/cs3org/reva@v0.1.1-0.20200320145721-40c4334849c3/internal/grpc/services/authprovider/authprovider.go:107 > error authenticating user error="authsvc: error in Authenticate: error: not found: admin" pid=306 pkg=rgrpc traceid=b642fa5c569ce1a933958d5017dec137
```
Is there "bonus" stuff happening in test setup/teardown or... that is trying to do stuff with an `admin` user that does not exist?

**Comment by kiranparajuli589** [comment]
> OCIS tests run really fast!

**O**hMyGod **CI** **S**peed!  :rocket:

**Pull request merged**